vispy/scene/cameras/panzoom.py

```diff
@@ -152,22 +154,23 @@ def rect(self, args):
     @property
     def center(self):
         rect = self._rect
-        return 0.5 * (rect.left + rect.right), 0.5 * (rect.top +
-                                                      rect.bottom), 0
+        return (*rect.center, 0)
```

Comment:
Is this syntax python 2 compatible? We are dropping python 2 support in the next minor release but it'd be nice to not deliberately lose it for this small change.

Reply:
Indeed, it's not python2 compatible, it's been a long time since I've dropped python2...

Reply:
Yeah, it would be nice if you could explicitly do the first and second element for this small change. I don't like it, but this type of change wasn't really the thing I envisioned as breaking our python 2 support.